pretyping/nativenorm.ml

```diff
@@ -303,10 +303,10 @@ and nf_atom_type env sigma atom =
       let (mib,mip) = Inductive.lookup_mind_specif env (fst ind) in
       let nparams = mib.mind_nparams in
       let params,realargs = Array.chop nparams allargs in
+      let nparamdecls = Context.Rel.length (Inductive.inductive_paramdecls (mib,u)) in
       let pT = 
-	hnf_prod_applist env 
+        hnf_prod_applist_assum env nparamdecls
```

Comment:
I realize now that you had already fixed this in `vm_compute`. I should really make progress on https://github.com/coq/coq/projects/17 to avoid this kind of duplicate bugs.

Reply:
And I heard that this would make @mattam82 and CertiCoq happy as well.